test/PayID/pay-id-client-test.ts

```diff
@@ -30,37 +30,32 @@ const beneficiaryName = 'xpring'
 const invoiceHash = 'some_invoice_hash'
 const transasctionConfirmation = 'some_transaction_confirmation'
 
-describe('Pay ID Client', function(): void {
-  afterEach(function() {
+describe('Pay ID Client', function (): void {
+  afterEach(function () {
     // Clean nock after each test.
     nock.cleanAll()
   })
 
-  it('xrpAddressForPayID - invalid Pay ID', function(done): void {
+  it('xrpAddressForPayID - invalid Pay ID', function (done): void {
     // GIVEN a PayIDClient and an invalid PayID.
     const invalidPayID = 'xpring.money/georgewashington' // Does not start with '$'
     const payIDClient = new PayIDClient(XRPLNetwork.Test)
 
-    // WHEN an XRPAddress is requested for an invalid pay ID THEN an invalid payment pointer error is thrown.
+    // WHEN an XRPAddress is requested for an invalid pay ID THEN an invalid Pay ID error is thrown.
     payIDClient.addressForPayID(invalidPayID).catch((error) => {
-      assert.equal(
-        (error as PayIDError).errorType,
-        PayIDErrorType.InvalidPaymentPointer,
-      )
+      assert.equal((error as PayIDError).errorType, PayIDErrorType.InvalidPayID)
       done()
     })
   })
 
-  it('xrpAddressForPayID - successful response - match found', async function() {
+  it('xrpAddressForPayID - successful response - match found', async function () {
     // GIVEN a PayID client, valid PayID and mocked networking to return a match for the PayID.
-    const payID = '$xpring.money/georgewashington'
+    const payID = 'georgewashington$xpring.money'
```

Comment:
Just a thought: would it make sense to create a global variable like `VALID_PAY_ID` and use it everywhere that we're using a valid payID?  Shortens the code a little and makes it obvious what is and isn't a valid PayID.  Otherwise looks great to me.